app/Models/Permission.php

```diff
@@ -115,125 +115,332 @@ class Permission extends Model implements Validatable
     ];
 
     /**
-     * All the permissions available on the system. You should use self::permissions()
-     * to retrieve them, and not directly access this array as it is subject to change.
+     * All the permissions available on the system. Use Permission::permissionTabs() or Permission::permissions()
      *
-     * @see Permission::permissions()
+     * @return array<int, array{
+     *      name: string,
+     *      description: string,
+     *      icon: string,
+     *      checkboxList: array{
+     *          name: string,
+     *          columns: int,
+     *          options: array<int, array{
+     *              name: string,
+     *              label: string,
+     *              description: string,
+     *          }>
+     *      }
+     *  }>
      *
-     * @var array<array-key, array{
-     *     description: string,
-     *     keys: array<array-key, string>,
-     * }>
+     * @see Permission::permissionTabs()
      */
-    protected static array $permissions = [
-        'websocket' => [
-            'description' => 'Allows the user to connect to the server websocket, giving them access to view console output and realtime server stats.',
-            'keys' => [
-                'connect' => 'Allows a user to connect to the websocket instance for a server to stream the console.',
-            ],
-        ],
-
-        'control' => [
-            'description' => 'Permissions that control a user\'s ability to control the power state of a server, or send commands.',
-            'keys' => [
-                'console' => 'Allows a user to send commands to the server instance via the console.',
-                'start' => 'Allows a user to start the server if it is stopped.',
-                'stop' => 'Allows a user to stop a server if it is running.',
-                'restart' => 'Allows a user to perform a server restart. This allows them to start the server if it is offline, but not put the server in a completely stopped state.',
+    public static function permissionTabs(): array
+    {
+        return [
+            [
+                'name' => 'Console',
+                'description' => trans('server/users.permissions.control_desc'),
+                'icon' => 'tabler-terminal-2',
+                'checkboxList' => [
+                    'name' => 'control',
+                    'columns' => 2,
+                    'options' => [
+                        [
+                            'name' => 'Console',
+                            'label' => 'console',
+                            'description' => trans('server/users.permissions.control_console'),
```

Comment:
The `description` could be dynamically set, right?
The trans key is always `server/users.permissions.` + the group (e.g. `control`) + `_` + permission name (e.g. `console`).

Also if we do that we don't need `name` and `label` but use a simple string array for the options.
Example:
```php
'options' => ['console', 'start', 'restart', 'stop'],
```
Display name would be something like `Str::headline($key)`, and description `trans('server/users.permissions.' . $group . '_' . $key)`.